websites/W/WebtoonXYZ/dist/metadata.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Webtoon",
```

Comment:
service name and folder name must be the same

Reply:
Oh I didn't know that, I'll fix it